logging/eventrouter/eventrouter.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: v4m-eventrouter
-  namespace: "logging"
+  name: eventrouter-v4m
```

Comment:
I lean towards making the fewest changes necessary.  With that in mind, do we really need to change _every_ references of eventrouter (and/or v4m-eventrouter) to eventrouter-v4m.  I  accept that inverting things in the the pod name gets us over the Fluent Bit hurdle, but I'm not sure there's value in extending that non-standard naming to everything related to eventrouter.  Do you feel that the value of keeping things consistent within the eventrouter "bubble" outweighs the value of keeping things consistent within the larger v4m "bubble"?  Please give that some thought and let's discuss. And if there are _technical_ reasons why resource names must match please call those out.  For example, do the ServiceAccount and ClusterRole names have to match each other and/or the pod name?

Reply:
That makes sense.

I just changed it to where I only change the metadata name (and anything that uses it) to be **eventrouter-v4m** and reverted all other instances back to just **eventrouter**.  It should show up on the next update.